pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.fr-fr.md

```diff
@@ -1,20 +1,49 @@
 ---
 title: Configuring multi-attach persistent volumes with OVHcloud NAS-HA
+slug: configuring-multi-attach-persistent-volumes-with-ovhcloud-nas-ha
 excerpt: 'Find out how to configure a multi-attach persistent volume using OVHcloud NAS-HA'
-updated: 2023-06-30
+section: Storage
```

Comment:
Please revert to keep only `title:`, `excerpt:`and `updated:`
The other tags are deprecated.